pkg/printers/internalversion/printers.go

```diff
@@ -73,6 +73,8 @@ const (
 
 	// nodeLabelRole specifies the role of a node
 	nodeLabelRole = "kubernetes.io/role"
+
+	staleDescription = "Stale field indicates whether status.observedGeneration matches generation of the object."
```

Comment:
status.observedGeneration is not a standard field... I would not expect to teach printers about this

Reply:
I taught only workload printers that have the field. Printers already seem to differ by type.

Reply:
@liggitt we already print artificial information about pods, for example. In this particular case I think it's valuable to easily tell from looking at get whether the status matches the spec or not at this point in time. In most cases controllers managed to update the status before user notices, but in those cases when they don't we answer questions such as "why my change isn't yet reflected", and having the answer to that question in the status of the workloads controllers seems reasonable.

Reply:
@liggitt I'd like to revisit our discussion in this topic, if you look at https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/controllers.md namely point 10 which is discussing the `status.observedGeneration`. I'm not forcing one way or the other of presenting that information but I'd like to be able to show a user an explicit information whether a controller acted on a resource I'm displaying. In 99% of cases it will be negligible but that 1% chance that you might hit that millisecond window between object modification and when a controller reacts when the spec will differ from the status. Most likely, the status will reconcile to a proper one before you invoke another `kubectl get` but you might confuse user during that short period of time. Having that information explicit in the get output is in my opinion reasonable, the only question is how to best express that?